code/style.css

```diff
@@ -1,4 +1,305 @@
+     font-size: 16px;
+     padding-bottom: 10px;
+     color: coral;
+ }
+
+/************* SMALL NEWS SECTION ***************/
+.small-news{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    flex-wrap: wrap;
+}
+
+.sn-card{
+    margin: 10px;
+    margin-left: 20px;
+    padding: 10px;
+    display: flex;
+    flex-direction: column;
+    justify-items: center;
+    justify-content:flex-start; /*content upp i början*/
+    text-align:center;  
+}
+
+.sn-img-container {
+    overflow: hidden;
+    padding-bottom: 20px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 
+                0 6px 20px 0 rgba(0, 0, 0, 0.19);
+   }
+
+.sn-img-container img {
+    height: 100%;
+    width: 100%;
+    object-fit: cover; 
+}
+
+.sn-card h3 {
+    font-size: 18px;
+    padding: 5px;
+    color: coral;
+}
+
+.sn-card p{
+    padding: 10px;
+    font-size: 12px;
+}
+
+
+
+/******** MEDIA QUERIES ***********/
+@media (min-width: 768px) {
+    /* Tablet view */
+
+/***** HEADER ******/
+    .header{
+        justify-content: space-between;
+    }
+
+    .header-title {
+        display: none;
+    }
+
+    .navigation {
+        display: inline-block;
+    }
+
+    .mobile-hamburger{
+        display:none; 
+    }
+
+    /************** BIG NEWS SECTION ******************/
+    .bn-card{
+        flex-direction: row;
+        justify-content: space-around;
+    }
+
+    .img-container {
+        width: 50%;
+    }
+
+    .bn-text {
+        width: 50%;
+        padding-left: 10px;
+    }
+
+    /************* SMALL NEWS SECTION ***************/
+
+    .small-news {
+        padding: 10px 30px;
+    }
+
+    .small-news{
+        flex-direction: row;
+        flex-wrap: wrap;
+    }
+   .sn-card{
+       max-width: 45%;
+       flex-direction: column; 
+   }
+
+    .sn-card h3 {
+        font-size: 20px;
+    }
+    .sn-card p {
+        width: 100%;
+    }
+    .sn-img-container {
+        width: 100%;
+    }
+
+}
+
+
+@media (min-width: 992px) {
+    /* Desktop view */
+    
+    /***** HEADER ******/
+    .header{
+        justify-content: space-between;
+    }
+
+    .logo {
+        width: 110px;
+    }
+
+    .header-title {
+        display: inline-block;
+        font-size: 30px;
+        padding-left: 150px;
+    }
+      
+    .navigation {
+        display: inline-block;
+    }
+
+    .mobile-hamburger{
+        display:none; 
+    }
+
+    .navigation a {
+        transition: all 0.8s ease 0s;
+    }
+
+    .navigation a:hover {
+     color: coral;
+     text-decoration: underline;
+    }
+
+    /************** BIG NEWS SECTION ******************/
+
+    .bn-text {
+        width: 45%;
+    }
+
+    .bn-text h1 {
+        font-size: 30px;
+        margin-bottom: 10px;
+    }
+
+    
+    /************* SMALL NEWS SECTION ***************/
+
+    .small-news{
+        flex-direction: row;
+        flex-wrap: wrap;
+        padding-left: 20px;
+    }
+
+   .sn-card{
+       max-width: 20%;
+       flex-direction: column;
+    }
```

Comment:
I think your cards look so good on all widths! If you want to have the cards a little bigger that's fine (e.g.` max-width: 30%;`)